src/components/PlanCard/collapsible-details.js

```diff
@@ -50,7 +50,7 @@ const FeatureDetails = (props) => {
 
   return (
     <FeatureDetailsWrapper>
-      <div onClick={function () {
+      <div  onClick={function () {
```

Comment:
Please do not change anything else in the code. Specifically, avoid making whitespace changes or other lint-related modifications.